sinfonier-web/app/controllers/Topologies.java

```diff
@@ -421,6 +421,13 @@ public static void doImport() throws SinfonierException {
       response.status = c400.getCode();
       renderJSON(c400.toGSON());
     }
+    catch (Exception e) {
```

Comment:
The specific exceptions has been catched, this point is the top of the bad responses. This method is tracked by ajax call, so I need to return always a valid json